pr-best-practices.md

```diff
@@ -0,0 +1,11 @@
+# PR Best Practices
+
+## Keep them small
+
+Keep your pull requests as small as possible. We should be looking for ways to split up our work into the smallest units that are sensible to merge. Some things we should be thinking:
+
+- *"Are there any small code refactors I can do as a preliminary PR into edge that will make completing this feature a trivial task?"*
+- *"Are there sub-features of this Jira ticket that can be deployed in isolation and have benefit to the product before the rest of my work is done?"*
+- *"Perhaps I should merge lots of small pull requests into a separate branch for my feature and have those reviewed individually, and then merge that into edge once it's tested."*
```

Comment:
mega-de and heha got hit by this approach recently. You do still end up with a big code change in a single deploy.